### PR TITLE
Adjudication base page

### PR DESCRIPTION
**Pull request by mathurahravigulan**
Listed performances on adjudication page:
- get performances by event ID
- show school, group size, performance type, and title of piece in table

Started form page to add new adjudications:
- get performance details by performance ID and event ID

### CODE REVIEW

File: db/init_db.py [hunk 1]
```diff
@@ -8,17 +8,17 @@ def _init_postgres_db(app):
     migrate.init_app(app, db)
```
Comment (brandonleung) [inline db/init_db.py:8]: unnecessary changes in this file

File: frontend/src/components/App.jsx [hunk 2]
```diff
@@ -130,14 +132,16 @@ export default class App extends React.Component {
           {
             loading ? (<Loading />) : (
               <React.Fragment>
-                { user && <Header /> }
+                { user && <Header />}
```
Comment (brandonleung) [inline frontend/src/components/App.jsx:135]: revert

File: frontend/src/components/App.jsx [hunk 3]
```diff
@@ -98,12 +100,12 @@ const PrivateRoute = ({
   user,
   ...rest
 }) => (
-  <Route
-    path={path}
-    render={props => (user ? <Component {...props} />
-      : <Redirect to={{ pathname: '/', state: { from: props.location }}} />)}
-    {...rest} />
-);
+    <Route
```
Comment (brandonleung) [inline frontend/src/components/App.jsx:103]: revert

File: frontend/src/components/adjudications/AdjudicateForm.jsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+import { getPerformance } from '../../api/PerformanceAPI';
+import humps from 'humps';
+
+export default function AdjudicateForm(props) {
+    const { match: { params: { eventId, performanceId }}} = props;
+    const [loading, setLoading] = useState(true)
+    const [adjudications, setAdjudications] = useState({}) 
+    const [performanceValues, setPerformancesValues] = useState({}) 
+
+//according to docs, componentDidMount() is similar to useEffect(() => {});
```
Comment (brandonleung) [inline frontend/src/components/adjudications/AdjudicateForm.jsx:19]: nitpick: align the comments and closing bracket for this function, and indent the `.then`s and corresponding brackets

File: frontend/src/components/adjudications/AdjudicateForm.jsx [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+        });
+
+        getAdjudications(performanceId)
+        .then(({data}) => {
+            setAdjudications(data)  
+            setLoading(false)
+        });
+  }, []); //added the empty array so that it will only be called after the component mounts
+
+    console.log("performance values are:")
```
Comment (brandonleung) [inline frontend/src/components/adjudications/AdjudicateForm.jsx:33]: remove `console.log` statements

File: frontend/src/components/adjudications/AdjudicateForm.jsx [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+    console.log(performanceValues) //works!!
+    console.log("adjudications are:")
+    console.log(adjudications) //works!!
+
+    const [artisticMark, setartisticMark] = useState(0) //currentValues.artisticMark
```
Comment (brandonleung) [inline frontend/src/components/adjudications/AdjudicateForm.jsx:38]: camel case the function names: `setArtisticMark` not `setartisticMark`

File: frontend/src/components/adjudications/AdjudicateForm.jsx [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+          label: 'Choreography Award',
+          name: 'choreoAward',
+          value: 'choreoAward'
+        }
+      ];
+    //handle change of text fields
+    const handleArtisticScoreChange = (e) => {
+        const { name, value } = e.target;
+        const keys = ['artisticMark', 'technicalMark'];
+        setartisticMark(keys.includes(name) ? parseInt(value, 10) : value)
```
Comment (brandonleung) [inline frontend/src/components/adjudications/AdjudicateForm.jsx:61]: by default the radix should be 10 for `parseInt`

File: frontend/src/components/adjudications/AdjudicationSection.jsx [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+import humps from 'humps';
+import { useState, useEffect } from 'react';
+
+
+export default function AdjudicationsSection(props) {
+  const [loading, setLoading] = useState(true)
+  const [adjudications, setAdjudications] = useState({}) 
+  const [performances, setPerformances] = useState({}) 
+  const collectionName = `events/${eventId}/performances/1/adjudications`; //replace 1 with ${performanceId}
+  const [filteredPerformances, setFileterdPerformances] = useState([])
```
Comment (brandonleung) [inline frontend/src/components/adjudications/AdjudicationSection.jsx:17]: spelling lol

**Review by brandonleung — CHANGES_REQUESTED** [review]
only small changes - good work @ChidiNweze @mathurahravigulan !

File: frontend/src/components/adjudications/AdjudicateForm.jsx (hunk 6 above)
Comment (ChidiNweze) [reply frontend/src/components/adjudications/AdjudicateForm.jsx]: Fixed this for setArtisticMark and setTechnicalMark.

**Review by brandonleung — APPROVED** [review]
good work! good to merge - add a description to the PR saying what the changes are (the first task that you closed on notion)
